java/distance-measurement-analysis/src/main/java/com/esri/arcgisruntime/sample/distancemeasurementanalysis/MainActivity.java

```diff
@@ -64,14 +65,14 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // create a scene and add a basemap to it
     final ArcGISScene scene = new ArcGISScene();
-    scene.setBasemap(Basemap.createImagery());
+    scene.setBasemap(Basemap.createTopographic());
 
     // inflate views from layout
     mSceneView = findViewById(R.id.sceneView);
     mDirectDistance = findViewById(R.id.totalDistance);
     mHorizontalDistance = findViewById(R.id.horizontalDistance);
     mVerticalDistance = findViewById(R.id.verticalDistance);
-    Spinner mUnitSpinner = findViewById(R.id.units_spinner);
+    Spinner unitSpinner = findViewById(R.id.units_spinner);
     // set the scene to the view
     mSceneView.setScene(scene);
 
```

Comment:
to avoid the scene layer cutting into the terrain, we're using an altitude offset on the scene layer:

``` java
    sceneLayer.setAltitudeOffset(1); // offset for visual purposes
```